packages/ember-glimmer/lib/components/link-to.js

```diff
@@ -594,17 +594,19 @@ const LinkComponent = EmberComponent.extend({
   }),
 
   transitioningIn: computed('active', 'willBeActive', function computeLinkToComponentTransitioningIn() {
-    let willBeActive = get(this, 'willBeActive');
-    if (typeof willBeActive === 'undefined') { return false; }
-
-    return !get(this, 'active') && willBeActive && 'ember-transitioning-in';
+    if (get(this, 'willBeActive') === true && !get(this, 'active')) {
```

Comment:
they might return
`get(this, 'active')` = > false, undefined, string
`get(this, 'willBeActive')` => true, false, undenfined (I am removing this case)